app/controllers/api/v3/statements_controller.rb

```diff
@@ -1,8 +1,21 @@
 module API
   module V3
     class StatementsController < BaseController
-      def show = head(:method_not_allowed)
-      def index = head(:ok)
+      def index
+        render json: StatementSerializer.render(statements_query.statements, root: "data")
+      end
+
+      def show
+        render json: StatementSerializer.render(statements_query.statement(id: params[:id]), root: "data")
```

Comment:
Having to add `root: "data"` everywhere isn't the prettiest 😞 

[Doesn't look like there's any way around it either](https://github.com/procore-oss/blueprinter/issues/253).

Reply:
Maybe we can add a wrapper for this method in the base controller, and pass in the serialiser/params to avoid calling this everywhere?

Agree though its not great, and as @mooktakim mentioned in another conversation keep our options open to using other serialisers if we feel it might be a problem

Reply:
could we also add strong params, something similar to:
```
def statement_params
  params.permit(:id)
end
```

Reply:
Also if we stick with BluePrinter and extend the query objects so they're modular and chainable, we could put th serialization there make this controller a little less involved, something more like:

```ruby
render(json: Statements::Query.new.for_lead_provider(current_lead_provider).serialize(:v3))
```

Reply:
Update code with strong params 👍

Reply:
`root: "data"` definitely not nice. I'm actually leaning towards going back to `jsonapi-serializer` 😂

Reply:
I tried to DRY up the code a bit with `to_json` method